pkg/deploy/rp.go

```diff
@@ -377,6 +379,12 @@ echo "$MDMPRIVATEKEY" >/etc/mdm/key.pem
 chown -R 1000:1000 /etc/mdm
 chmod 0600 /etc/mdm/key.pem
 
+mkdir -p /etc/mdsd
+echo "$MDSDCERTIFICATE" >/etc/mdsd/cert.pem
+echo "$MDSDPRIVATEKEY" >/etc/mdsd/key.pem
+chown -R 1000:1000 /etc/mdsd
+chmod 0600 /etc/mdsd/key.pem
```

Comment:
I think I provided a bad steer on this - I can't think of a reason why we wouldn't dynamically pick this out of the service key vault using `f.env.GetCertificateSecret`, just like, say, the `rp-server` cert and key?

May need updates to the documentation to get this key/cert into the svc keyvault up front.